client/nginx.go

```diff
@@ -646,6 +716,15 @@ func (client *NginxClient) getServerZones() (*ServerZones, error) {
 	return &zones, err
 }
 
+func (client *NginxClient) getStreamServerZones() (*StreamServerZones, error) {
+	var zones StreamServerZones
+	err := client.get("stream/server_zones", &zones)
+	if err != nil {
```

Comment:
If the stream {} block is not present in the NGINX Plus configuration, NGINX Plus will return 404 "stream not configured (StreamNotConfigured)" for any /stream API calls. If that is the case, our client must recognize that and not fail!

Reply:
done